### PR TITLE
[merged] Add support for deleting containers

### PR DESCRIPTION
**Pull request by rhatdan**
We want to be able to delete containers from the system, using the
new atomic containers delete call.

atomic containers delete --all

Comes in handy.

### CODE REVIEW

**Comment by rhatdan** [comment]
@giuseppe PTAL

File: Atomic/containers.py [hunk 1]
```diff
@@ -194,3 +213,38 @@ def _filter_include_container(self, container_info):
...
+                if c["Type"] == "system":
+                    sys_targets.append(c["Id"])
+                else:
+                    docker_targets.append(c["Id"])
+        else:
+            for c in self.args.containers:
+                if self._inspect_container(c):
+                    docker_targets.append(c)
+                else:
+                    sys_targets.append(c)
```
Comment (giuseppe) [inline Atomic/containers.py]: we look up system containers first then try the docker version.  I think here we should do:
```
if self.syscontainers.get_system_container_checkout(self, name):
    sys_targets.append(c)
else:
    docker_targets.append(c)
```

Comment (rhatdan) [inline Atomic/containers.py]: You mean?
```
if self.syscontainers.get_system_container_checkout(c):
    sys_targets.append(c)
else:
    docker_targets.append(c)
```

Comment (giuseppe) [inline Atomic/containers.py]: yes, sorry :)

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 2fb55c7

**Pull request closed without merging**